camera.js

```diff
@@ -1389,6 +1437,9 @@ s.camera=function(x,e,cn,tx){
                                             e.fn()
                                         break;
                                         case /T[0-9][0-9]-[0-9][0-9]-[0-9][0-9]./.test(d):
+                                            if( e.details.gensubs==1 ) {
```

Comment:
`e.details.gensubs==1` should be `e.details.gensubs==='1'` for consistency and because it is a string anyway.